.vscode/launch.json

```diff
@@ -73,7 +73,7 @@
                     "enabled": true
                 }
             },
-            "justMyCode": false,
+            "justMyCode": true,
```

Comment:
Any specific reason or this change?

Reply:
Yes, for vscode debugger to not step in to sdk/clr.